en/releases/README.md

```diff
@@ -1,5 +1,5 @@
 # Releases
 
-In depth release notes of what changed and how to adapt and make best use.
+Notes explaining changes in each release.
```

Comment:
Note the difference. Feel free to add back "how to adapt to best use" when we have real docs in place :-)